src/appleseed/renderer/modeling/project/projectfilewriter.cpp

```diff
@@ -925,4 +928,48 @@ bool ProjectFileWriter::write(
     return true;
 }
 
+bool ProjectFileWriter::write(
+        const Project&  project,
+        const char*     filepath,
+        const int       options)
+{
+    if (!(options & ProjectFileWriter::PackedProject))
+        return write_project_file(project, filepath, options);
+    else
+    {
+        bf::path temp_project_filepath;
+
+        try
+        {
+            const bf::path project_path(filepath);
+            temp_project_filepath = project_path.parent_path() /
+                "temp.unpacked" / project_path.filename().replace_extension(".appleseed");
+
+            bf::create_directory(temp_project_filepath.parent_path());
+
+            bool success = write_project_file(project,
```

Comment:
Make success const.

Fix indentation, I suggest this:

    const bool success =
        write_project_file(
            project,
            temp_project_filepath.c_str(),
            options | ProjectFileWriter::CopyAllAssets);